advanced_functionality/causal-inference/causal-inference-container.ipynb

```diff
@@ -0,0 +1,588 @@
+{
```

Comment:
<p>Add <code>predictor.delete_model()</code>  before <code>predictor.delete_endpoint()</code>  so we clean up the model as well as the endpoint. Make sure to perform delete_model before delete_endpoint or else an error will be thrown.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3130/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>